staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -385,28 +385,60 @@ func (*Mounter) List() ([]MountPoint, error) {
 	return ListProcMounts(procMountsPath)
 }
 
+// statx support since Linux 4.11, glibc 2.28
+// refer: https://man7.org/linux/man-pages/man2/statx.2.html
+var errNotSupport = errors.New("The statx syscall is not supported. At least Linux kernel 4.11 is needed")
+
+func statx(file string) (unix.Statx_t, error) {
+	// TODO add STATX_ATTR_MOUNT_ROOT support
```

Comment:
What does TODO mean in this context?

Reply:
It's useless comment here, sorry.  removed.